app-backup/rsnapshot/metadata.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!--maintainer-needed-->
+  <maintainer type="person">
+    <email>Sergiy.Borodych@gmail.com</email>
+    <name>Sergiy Borodych</name>
+  </maintainer>
```

Comment:
Please re-add proxy-maint.

Reply:
Yes, missed that.
Thank you for the point.
Added.